cpp/ql/lib/experimental/quantum/Language.qll

```diff
@@ -86,6 +86,27 @@ module GenericDataSourceFlowConfig implements DataFlow::ConfigSig {
   }
 }
 
+module GenericDataSourceFlow = TaintTracking::Global<GenericDataSourceFlowConfig>;
+
+private class ConstantDataSource extends Crypto::GenericConstantSourceInstance instanceof Literal {
```

Comment:
Can you test this with MRVA or DCA before merging? I worry the performance impact will be severe from this.